backend/src/test/java/com/example/backend/organization/OrganizationIntegrationTests.java

```diff
@@ -28,13 +34,24 @@ class OrganizationIntegrationTests {
     @Autowired
     private OrganizationRepo organizationRepo;
     @Autowired
+    private MongoUserRepository mongoUserRepository;
+    @Autowired
     private ObjectMapper mapper;
     private Organization dummyOrganization;
+    private MongoUser dummyUser;
     private String jsonOrganization;
+    private String jsonMongoUser;
     private String jsonWithoutId;
+    private String jsonMongoUserWithoutId;
+
 
     @BeforeEach
     void setUp() throws Exception {
+        dummyUser = new MongoUser("564", "Carina", "Carina1", Role.ADMIN);
+        jsonMongoUser = mapper.writeValueAsString(dummyUser);
+        jsonMongoUserWithoutId = """
+                {"username":"Carina","role":"ADMIN"}
+                """;
```

Comment:
Why do you create on one hand a object and map it and on the other hand you write the userWithoudId manually?
Is this the UserDTO, isn't it?

Reply:
I think the jsonMongoUserWithoutId is the userDTO that you use?! 
Than you can do something like this:
```suggestion
        dummyUser = new MongoUser("564", "Carina", "Carina1", Role.ADMIN);
        jsonMongoUser = mapper.writeValueAsString(dummyUser);
        
        dummyUserDTO = new UserDTP(dummyUser) // map dummy user to userDTO
        jsonMongoUserWithoutId = objectMapper.writeValueAsString(dummyUserDTO)
```